components/crypto_dot_com/browser/buildflags/buildflags.gni

```diff
@@ -0,0 +1,5 @@
+import("//build/config/features.gni")
+
+declare_args() {
+  crypto_dot_com_enabled = is_mac || is_linux || is_win
```

Comment:
Follow up: This should be in a runtime flag instead of a build flag, but that can be done with the other refactoring you'll be doing next.